instrumentation-genai/opentelemetry-instrumentation-google-genai/README.rst

```diff
@@ -60,13 +60,23 @@ Make sure to configure OpenTelemetry tracing, logging, metrics, and events to ca
         model="gemini-1.5-flash-002",
         contents="Write a short poem on OpenTelemetry.")
 
+Enabling experimental semconvs
+******************************
+
+Set the environment variable `OTEL_SEMCONV_STABILITY_OPT_IN` to `gen_ai_latest_experimental` to 
+emit the latest experimental version of GenAI conventions and do not emit the old ones (v1.36.0 or prior).
```

Comment:
I would omit the "old ones" reference, 1.38 could be the old one very soon as these are moving very fast this days :)